pages/landing/hipaa/pricing/index.html

```diff
@@ -93,6 +82,13 @@
       <p>
         However, we offer a 30-day money back guarantee, so if for any reason you're not happy after signing up, we'll let you out of the annual subscription and refund 100% of your money in the first 30 days.
       </p>
+  - question: How does early access work?
+    answer: |
+      <p>Early access subscribers to Gruntwork HIPAA will get the full benefit of the Gruntwork CIS offering available today, in addition to direct HIPAA-compliance audit support and the lowest overall cost for the HIPAA-compliance Reference Architecture.</p><p>It can be helpful to think of this program as occuring in 3 distinct phases:</p><ul><li><b>Phase 1: CIS Deployment</b> Early access subscribers will receive a CIS Benchmark compliant Reference Architecture. The CIS Reference Architecture comes complete with CIS hardened base images and already satisfies many of HIPAA's requirements.</li><li><b>Phase 2: Gruntwork completes HIPAA-compliance</b> The Gruntwork team works to implement the final changes to make the Reference Architecture and Reference Applications 100% HIPAA-compliant. See <a href="/hipaa-compliance-on-aws/technical-details/#supported-infrastructure-features">this table for details.</a> As soon as this work is complete, Gruntwork will update your CIS Reference Architecture to be 100% HIPAA-compliant at no extra charge.</li><li><b>Phase 3: Audit assistance</b> Early access subscribers will recieve direct support from Gruntwork, who will go through a third-party HIPAA-compliance audit alongside you and fix any issues the auditor may find.</li></ul>
```

Comment:
```suggestion
      <p>Early access subscribers to Gruntwork HIPAA will get the full benefit of the Gruntwork CIS offering available today, in addition to direct HIPAA-compliance audit support and the lowest overall cost for the HIPAA-compliance Reference Architecture.</p><p>It can be helpful to think of this program as occuring in 3 distinct phases:</p><ul><li><b>Phase 1: CIS Deployment.</b> Early access subscribers will receive the CIS Reference Architecture on day 1, which is compliant with the CIS AWS Foundations Benchmark out-of-the-box. You can use this immediately to set up your pre-prod environments in AWS.</li><li><b>Phase 2: Gruntwork completes HIPAA-compliance.</b> The Gruntwork team works to implement the final changes to make the Reference Architecture and Reference Applications 100% HIPAA-compliant. See <a href="/hipaa-compliance-on-aws/technical-details/#supported-infrastructure-features">this table for the work we'll be doing.</a> As an early access subscriber, you'll get access to each change as soon as it's available in the form of a versioned release. Also, as soon as all the HIPAA work is complete, Gruntwork will deploy a HIPAA-compliant prod environment for you at no extra charge.</li><li><b>Phase 3: Audit assistance.</b> As you go through a HIPAA-compliance audit with a third-party, Gruntwork will provide assistance, and quickly fix any issues the auditor identifies.</li></ul>
```